bcdaworker/main.go

```diff
@@ -204,17 +204,28 @@ func writeBBDataToFile(bb client.APIClient, acoID string, beneficiaryIDs []strin
 
 	w := bufio.NewWriter(f)
 	errorCount := 0
-	totalBeneIDs := float64(len(beneficiaryIDs))
+	totalBeneIDs := float64(len(cclfBeneficiaryIDs))
 	failThreshold := getFailureThreshold()
-
-	for _, beneficiaryID := range beneficiaryIDs {
-		pData, err := bbFunc(beneficiaryID, jobID)
+	db := database.GetGORMDbConnection()
+	defer db.Close()
+	for _, cclfBeneficiaryID := range cclfBeneficiaryIDs {
+		var cclfBeneficiary models.CCLFBeneficiary
+		db.First(&cclfBeneficiary, cclfBeneficiaryID)
+		blueButtonID, err := cclfBeneficiary.GetBlueButtonID(bb)
+		if err != nil {
+			log.Error(err)
+			errorCount++
+			appendErrorToFile(acoID, responseutils.Exception, responseutils.BbErr, fmt.Sprintf("Error retrieving BlueButton ID for cclfBeneficiary %s", cclfBeneficiaryID), jobID)
+		}
+		cclfBeneficiary.BlueButtonID = blueButtonID
+		db.Save(&cclfBeneficiary)
```

Comment:
Just to confirm, this line fully address this comment, correct: 
https://github.com/CMSgov/bcda-app/pull/252/files#r280442154

Do we always need to call `db.Save()` here?  What if the `BlueButtonID` already exists for the `CCLFBeneficary`?  The extraneous call to save is harmless, but probably not necessary if the ID is already there.